presto-native-execution/presto_cpp/presto_protocol/tests/PlanFragmentTest.cpp

```diff
@@ -25,11 +25,11 @@ using namespace facebook::presto;
 
 namespace {
 std::string getDataPath(const std::string& fileName) {
-  static const std::string kFbcode = "fbcode";
-
   std::string currentPath = fs::current_path().c_str();
-  if (boost::algorithm::ends_with(currentPath, kFbcode)) {
-    return currentPath.substr(0, currentPath.length() - kFbcode.length()) + "third-party/presto_cpp/presto_protocol/tests/data/" + fileName;
+  if (boost::algorithm::ends_with(currentPath, "fbcode")) {
+    return currentPath +
+        "/github/presto-trunk/presto-native-execution/presto_cpp/presto_protocol/tests/data/" +
+        fileName;
```

Comment:
I wonder if there is a better way to get a relative path for the test?
Like we are running a test from the ".../tests/" folder.
If we could access it and avoid that code fork which would probably not work on an external user's inveronment...
Could we also not use the data file, but compile it into the test and in the test stage write it as a tmp file?